test/language/default-parameters/scope.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 13.3.3.7
+description: >
+  Expressions as parameter default values share the top level context only.
```

Comment:
This test could use more formal justification. Something like:

    > 9.2.12 FunctionDeclarationInstantiation(func, argumentsList)
    >
    > 27. If hasParameterExpressions is false, then
    >     [...]
    > 28. Else,
    >     a. NOTE A separate Environment Record is needed to ensure that
    >        closures created by expressions in the formal parameter list do
    >        not have visibility of declarations in the function body.
    >     b. Let varEnv be NewDeclarativeEnvironment(env).
    >     c. Let varEnvRec be varEnv’s EnvironmentRecord.
    >     d. Set the VariableEnvironment of calleeContext to varEnv.